src/iceberg/pending_update.h

```diff
@@ -28,48 +28,67 @@
 
 namespace iceberg {
 
-/// \brief Base class for table metadata changes using builder pattern
```

Comment:
I don't prefer you to delete the context that you're using builder pattern. This is the reason why the ctors of this class are in the protected region, right?

Reply:
ack